doc/api.rst

```diff
@@ -31,6 +31,7 @@ mne_bids
    get_anat_landmarks
    update_anat_landmarks
    get_head_mri_trans
+   template_to_head
```

Comment:
There's a bit of an inconsistency in the naming here. Maybe the new function should be called `template_to_head_trans()`?

Reply:
It transforms the instance (could be an epoch but it's associated with `read_raw_bids` so I think `raw` is fine) so I think it shouldn't include the word `trans` because then someone might miss that it transforms the object, the `trans` is just secondary